offline_docs.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# Modify generated docs in a particular directory to work offline.
+
+date
+
+USAGE="Usage: offline_docs.sh [-d <dir-name>] [-r <dir-name>] [-z] [-o] [-h]."
```

Comment:
I'm guessing you plan to add more help content here? 

Might be nice to mention that the script creates a zip file called elastic-docs.zip, or give the user the option of setting the name (nice-to-have).